File: crates/ctxctl/tests/fixtures/sample.py
"""Fixture module used by ctxctl integration tests."""

import math


class Point:
    """A point in 2D space."""

    def __init__(self, x: float, y: float) -> None:
        self.x = x
        self.y = y

    def norm(self) -> float:
        return math.sqrt(self.x * self.x + self.y * self.y)


def add(a: int, b: int) -> int:
    """Adds two integers."""
    return a + b

File: crates/ctxctl/tests/fixtures/sample.rs
//! Fixture module used by ctxctl integration tests.

/// Adds two numbers.
pub fn add(a: i32, b: i32) -> i32 {
    a + b
}

struct Point {
    x: f64,
    y: f64,
}

impl Point {
    fn norm(&self) -> f64 {
        (self.x * self.x + self.y * self.y).sqrt()
    }
}

pub const ANSWER: i32 = 42;
